docs/providers.rst

```diff
@@ -1142,7 +1143,7 @@ SCOPE:
     https://developers.pinterest.com/docs/api/overview/#scopes
 
 QuickBooks
-------
+----------
```

Comment:
My editor was complaining about a bunch of warnings and errors in this file so I tidied them up